styles/main.css

```diff
@@ -117,7 +117,7 @@ footer hr {
     width: 50px;
     background-size: contain; 
     position: absolute;
-    right: 7px;
+    right: -7px;

```

Comment:
Using -ve value here pushes things to the right as requested by the lab.  +ve pushes things to the left.